src/Microsoft.Azure.ServiceBus/Extensions/MessageInterOpExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Extensions
```

Comment:
I deliberately chose extensions for the Uber class thinking if we add any other extensions that are not related to Interop and not related to message, we might be able to put them under the same extensions namespace